src/content/core/okteto-variables.mdx

```diff
@@ -7,36 +7,259 @@ id: okteto-variables
+
+:::
+
+
+### Configuring dynamic endpoints
+
+In this example, we show how `$OKTETO_ENV` can be used to create dynamic endpoints for external resources within your dev environment.
+
+If you are not familiar yet with the concept of Okteto external resource, image a component that is not part of the deployment of your dev environment, but that you need to interact with. For example, a database, a message queue, or an S3 bucket. If you need more details on this feature, check out [External Resources](/docs/development/external-resources).
+
+External resources are defined in the Okteto manifest and have a name and a URL. We understand that often, when using cloud providers, the resource URLs are generated dynamically when the infrastructure is provisioned. In this example, we make the S3 bucket name available to the rest of the deployment steps.
+
+```yaml
+## TODO: write example
+```
+
+
+## Setting Okteto Variables
+
+This section will show a brief example of how to set each type of Okteto Variable.
+
+### The --var flag
+
+The `--var` flag is useful to specify ad-hoc variables when running a command like [deploy](/docs/reference/okteto-cli#deploy) or [preview deploy](/docs/reference/okteto-cli#deploy-3)
+
+Example:
+
+```console
+$ okteto deploy --var PORT=4000
+```
+
+```yaml
+deploy:
+  image: "node:${TAG}"
+  commands:
+    - name: deploy
+      command: NODE_ENV=${ENV} make deploy
+```
+
+### Catalog
+
+When you deploy an application from the Okteto Dashboard or if your organization is using the Okteto Catalog, you can define project-specific variables. See [Catalog Variables](/docs/admin/catalog/#add-environment-variables) for more information.
+
+### Local Environment Variables
+
+Similar to the `--var` flag, local environment variables are useful when you need to specify ad-hoc variables when running a command like [deploy](/docs/reference/okteto-cli#deploy) or [preview deploy](/docs/reference/okteto-cli#deploy-3)
+
+Example:
+
+```console
+$ PORT=4000 okteto deploy
+```
+
+
+### The .env file
+
+You can set default values for any environment variables referenced in the okteto manifest in an environment file named `.env`.
+
+The `.env` should be placed in the same folder of the Okteto Manifest. The Okteto Manifest can also be stored in a special folder called `.okteto`. In that case, the `.env` should not be placed inside it, but in the parent directory.
+
+For example:
+
+```yaml
+# .env
+PORT=4000
+```
+
+```yaml
+# okteto.yml
+deploy:
+  commands:
+    - name: deploy
+      command: echo $PORT # -> 4000
+```
+
+:::tip
+
+The `.env` integration also supports parameter expansion to set default values and allowing other developers to override them.
+
+Example:
+
+```yaml
+# .env
+PORT=${PORT:-4000}
+```
+
+When running `PORT=5000 okteto deploy` the value of `PORT` will be `5000`.
 
 :::
 
-## Manage Okteto Variables from the Okteto dashboard
+
+### User and Admin variables
```

Comment:
They are different and created from different places, I would split this section in two

Reply:
agree, already changed 👍